### ISSUE TITLE
Photos to use 

### ISSUE THREAD

**Issue by Restuta** [Restuta/rcn.io#54]
http://www.gsbimages.com/Portfolio/Coppertown-Circuit-Race-2016/i-JNwKSsw/A

**Comment by Restuta** [comment]
![](http://media4.giphy.com/media/wS7S6BoIxeV20/giphy.gif)

**Comment by Me4ta** [comment]
I think we can add here links on Alex Chui and Alexandr Komlik photos.

**Comment by Restuta** [comment]
Yes, but we have to cherry-pick, above is a link to a particular photo

**Comment by Me4ta** [comment]
Yeah,  we can choose some of them for our needs later if we have time.
